app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<section class='recipes_index flex-items'>
+  <div class="recipe__description-container">
+    <h1>Recipes</h1>
+    <% if @recipes.count > 0%>
+      <%= render 'recipe'%>
+    <% else %>
+      <h2>There are no recipes yet</h2>
+    <% end %>
+    <div class="new__recipe-btn">
```

Comment:
- [Optional] Good job adding the link and the route for the recipe and food items. But currently, I have to click on the name of the recipe/food to go to the detail page. It would be wise to make the whole card a link to the detail page. This has a great advantage when it comes to the user experience.

![image](https://user-images.githubusercontent.com/69077061/233674669-32e40f2f-c93e-4fe5-9c1e-841991de14c5.png)